shell/browser/mac/in_app_purchase.mm

```diff
@@ -125,6 +130,7 @@ - (void)runCallback:(bool)isProductValid {
         FROM_HERE, base::BindOnce(std::move(callback_), isProductValid));
   }
   // Release this delegate.
+  [username_ release];
```

Comment:
This should be in the `-dealloc` method of this object, so that it's released no matter which way the object is deallocated.